doc/oil.txt

```diff
@@ -714,5 +717,35 @@ Mac:
 Windows:
     Oil supports the Windows Recycle Bin. All features should work.
 
+--------------------------------------------------------------------------------
+FILE CREATION                                                  *oil-file_creation*
+
+
+Oil defines `create_files_mode` to set the permission for new files. The mode
+follows the Posix numeric standard which use octal notation (base-8), however
+the variable must be set in decimal.
+
+    00700 user (file owner) has read, write, and execute permission
+    00400 user has read permission
+    00200 user has write permission
+    00100 user has execute permission
+    00070 group has read, write, and execute permission
+    00040 group has read permission
+    00020 group has write permission
+    00010 group has execute permission
+    00007 others have read, write, and execute permission
+    00004 others have read permission
+    00002 others have write permission
+    00001 others have execute permission
+    (from the open (2) man pages)
+
+An easy way to provide the right value for `create_files_mode` is using
+`tonumber(e [, base])` function with `base = 8`, for example:
+>lua
+    create_files_mode = tonumber("0644", 8)
+
+The default value is 420 in decimal or 0644 in octal
+
+
```

Comment:
I actually don't think we need this section. It's explaining:
1. what the octal file permissions mean
1. how to convert octal to decimal
1. the default value

Point 3 is already documented in the options section. Point 2 is obviated if we accept the permission in octal. And for point 1 I would argue that if you don't know what octal file permissions are, you probably shouldn't be changing this config value and I don't think the oil docs are the correct place to be learning about them.

Reply:
Ok, then I will remove it